apps/scan/frontend/src/app_root.tsx

```diff
@@ -128,7 +124,7 @@ export function AppRoot({
   const pollsInfo = pollsInfoQuery.data;
   const { pollsState } = pollsInfo;
 
-  if (!cardReader) {
+  if (!scannerStatus.isCardReaderConnected) {
```

Comment:
I wonder if we should surface `no_card_reader` as an auth logged out reason? Then we could just do all of this from the auth status object. That case has to be handled in the auth lib anyway

Reply:
[fcdec12](https://github.com/votingworks/vxsuite/pull/4315/commits/fcdec1219a52aa429076392d1a624c94fbc6b4d4)